runtime/lua/vim/lsp/diagnostic.lua

```diff
@@ -241,6 +241,25 @@ end
 -- Deprecated Functions {{{
 
 
+--- Display a deprecation notification to the user.
+--- @private
+---
+---@param name        string Deprecated function.
+---@param alternative string Preferred alternative function.
+---@param version     string Neovim version in which the deprecated function
+---                          will be removed.
+local function deprecate(name, alternative, version)
+    local message = name .. ' is deprecated'
+    if alternative then
+      message = message.. ', use ' .. alternative .. ' instead.'
+    else
+      message = message .. '.'
+    end
```

Comment:
```suggestion
    message = alternative and (message..', use '..alternative..' instead.') or (message..'.')
```

Reply:
could be simpler if we didn't end with a period. Messages generally don't end with a period.

(However, source-code comments _should_ use full punctuation.)

Reply:
Then let's remove them :)